src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -300,6 +300,7 @@ static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.VariableComponentAssignment(M
 static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.VariableComponentAssignment(Microsoft.CodeAnalysis.CSharp.Syntax.VariableComponentSyntax variableComponent, Microsoft.CodeAnalysis.SyntaxToken equalsToken, Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax value) -> Microsoft.CodeAnalysis.CSharp.Syntax.VariableComponentAssignmentSyntax
 static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.WhenClause(Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax condition) -> Microsoft.CodeAnalysis.CSharp.Syntax.WhenClauseSyntax
 static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.WhenClause(Microsoft.CodeAnalysis.SyntaxToken whenKeyword, Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax condition) -> Microsoft.CodeAnalysis.CSharp.Syntax.WhenClauseSyntax
+static Microsoft.CodeAnalysis.CSharp.SyntaxFacts.IsGlobalAttribute(Microsoft.CodeAnalysis.SyntaxNode node) -> bool
```

Comment:
Is this needed at the Compiler layer?  Could it not be internal at the workspace (or higher) level?